StandaloneMmPkg/Drivers/StandaloneMmIplPei/StandaloneMmIplPei.c

```diff
@@ -753,6 +795,44 @@ ExecuteMmCoreFromMmram (
     }
   }
 
+Done:
+  if (MmAccess != NULL) {
+    //
+    // Close all MMRAM ranges, if MmAccess is available.
+    //
+    for (Index = 0; Index < MmramRangeCount; Index++) {
+      Status = MmAccess->Close ((EFI_PEI_SERVICES **)GetPeiServicesTablePointer (), MmAccess, Index);
+      if (EFI_ERROR (Status)) {
+        DEBUG ((DEBUG_ERROR, "MM IPL failed to close MMRAM windows index %d - %r\n", Index, Status));
+        ASSERT (FALSE);
+        return Status;

```

Comment:
Should this really return instead of attempting to close and lock any potentially remaining windows?

Reply:
This is updated.